src/main/java/nextstep/subway/exception/handler/ApiHandler.java

```diff
@@ -0,0 +1,28 @@
+package nextstep.subway.exception.handler;
+
+import nextstep.subway.exception.PathFindException;
+import nextstep.subway.exception.SectionManagerException;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+@RestControllerAdvice
+public class ApiHandler {
+    @ExceptionHandler
+    public ResponseEntity PathFindExceptionHandler(PathFindException e){
+        return ResponseEntity.badRequest()
+            .build();
+    }
```

Comment:
예외 메시지도 같이 전달해 볼 수 있겠네요 
```suggestion
    @ResponseStatus(HttpStatus.BAD_REQUEST)
    @ExceptionHandler
    public ExceptionResponse PathFindExceptionHandler(PathFindException e){
        return new ExceptionResponse(e.getMessage());
    }
```

Reply:
ResponseEntity.badRequest()
 .body(e.getMessage); 로 메세징 처리